ghost/webmentions/lib/Mention.js

```diff
@@ -95,6 +95,12 @@ module.exports = class Mention {
         return this.#sourceFeaturedImage;
     }
 
+    /** @type {boolean} */
+    #verified;
```

Comment:
This was already implemented as part of https://github.com/TryGhost/Team/issues/2548

Reply:
Makes sense, thanks. Will remove

Reply:
Oops, thanks. It wasn't yet on my branch at the time.